tox.ini

```diff
@@ -122,6 +122,7 @@ commands =
   rm -rf docs/build
   bash -c "git ls-files docs/*.rst | while read file; do if ! grep -q :description: $file; then echo E: no description meta tag in $file; exit 1; fi; done"
   bash -c "cd docs && yarn && yarn build-bootstrap"
+  sphinx-build -Q -W -b linkcheck docs/source docs/build
```

Comment:
Maybe `-q` instead of `-Q`. Just in case, we shouldn't see a warning as we use -W but who known.